website/content/en/docs/olm-integration/quickstart-package-manifests.md

```diff
@@ -92,6 +92,70 @@ INFO[0000] operatorgroup "operator-sdk-og" deleted
 INFO[0001] operator "memcached-operator" uninstalled
 ```
 
+## Migrating packagemanifests to bundles
+
+In order to migrate packagemanifests to bundles, `operator-sdk pkgman-to-bundle` command can be used.
+
+As an example, consider the packagemanifests directory to have the following structure:
+
+```sh
+packagemanifests
+└── etcd
+    ├── 0.0.1
+    │   ├── etcdcluster.crd.yaml
+    │   └── etcdoperator.clusterserviceversion.yaml
+    ├── 0.0.2
+    │   ├── etcdbackup.crd.yaml
+    │   ├── etcdcluster.crd.yaml
+    │   ├── etcdoperator.v0.0.2.clusterserviceversion.yaml
+    │   └── etcdrestore.crd.yaml
+    └── etcd.package.yaml
+```
+
+Here, we have manifests for two versions of the `etcd` operator. The following command will generate bundles for each of these versions.
+
+```sh
+$ operator-sdk pkgman-to-bundle packagemanifests --output-dir etcd-bundle/
+INFO[0000] Packagemanifests will be migrated to bundles in bundle directory
+INFO[0000] Creating etcd-bundle/bundle-0.0.1/bundle.Dockerfile
+INFO[0000] Creating etcd-bundle/bundle-0.0.1/metadata/annotations.yaml
+```
+
+This will create output bundles in the directory `etcd-bundle`. The output directory will look like:
+
+```sh
+├── bundle-0.0.1
+│   ├── bundle.Dockerfile
+│   ├── manifests
+│   │   ├── etcdcluster.crd.yaml
+│   │   ├── etcdoperator.clusterserviceversion.yaml
+│   ├── metadata
+│   │   └── annotations.yaml
+│   └── tests
+│       └── scorecard
+│           └── config.yaml
```

Comment:
I think we said scorecard tests won't be added unless they exist in the packagemanifests dir, right?
```suggestion
│   └── metadata
│       └── annotations.yaml
```